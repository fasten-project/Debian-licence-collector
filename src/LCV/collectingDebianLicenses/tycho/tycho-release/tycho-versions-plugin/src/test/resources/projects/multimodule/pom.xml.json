{
    "checksum": "0dd6e868bc2d17bfe1d853bec1c45c07bdedb5d5e340f2f71e88f65a1c6a9da0",
    "file": "pom.xml",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 4,
    "package": "tycho",
    "path": "tycho/2.3.0-1/tycho-release/tycho-versions-plugin/src/test/resources/projects/multimodule/pom.xml",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/tycho/2.3.0-1/",
        "metric": {
            "size": 24640
        },
        "pts_link": "https://tracker.debian.org/pkg/tycho",
        "sloc": [
            [
                "java",
                78065
            ],
            [
                "xml",
                38107
            ],
            [
                "sh",
                64
            ],
            [
                "python",
                23
            ],
            [
                "makefile",
                10
            ]
        ],
        "suites": [
            "experimental"
        ],
        "vcs_browser": "https://salsa.debian.org/java-team/tycho",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/t/tycho/2.3.0-1/tycho-release/tycho-versions-plugin/src/test/resources/projects/multimodule/pom.xml",
    "stat": {
        "perms": "rw-r--r--",
        "size": 494,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "2.3.0-1"
}