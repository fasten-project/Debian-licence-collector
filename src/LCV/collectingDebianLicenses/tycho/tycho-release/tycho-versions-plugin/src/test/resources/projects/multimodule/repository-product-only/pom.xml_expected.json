{
    "checksum": "cd2b0726bca598c13552a0448d6b825751bf0fb76ad9e5369fad94b5d575b5d0",
    "file": "pom.xml_expected",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 4,
    "package": "tycho",
    "path": "tycho/2.3.0-1/tycho-release/tycho-versions-plugin/src/test/resources/projects/multimodule/repository-product-only/pom.xml_expected",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/tycho/2.3.0-1/",
        "metric": {
            "size": 24640
        },
        "pts_link": "https://tracker.debian.org/pkg/tycho",
        "sloc": [
            [
                "java",
                78065
            ],
            [
                "xml",
                38107
            ],
            [
                "sh",
                64
            ],
            [
                "python",
                23
            ],
            [
                "makefile",
                10
            ]
        ],
        "suites": [
            "experimental"
        ],
        "vcs_browser": "https://salsa.debian.org/java-team/tycho",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/t/tycho/2.3.0-1/tycho-release/tycho-versions-plugin/src/test/resources/projects/multimodule/repository-product-only/pom.xml_expected",
    "stat": {
        "perms": "rw-r--r--",
        "size": 287,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "2.3.0-1"
}